eve/workers/pod-builder/Dockerfile

```diff
@@ -26,4 +26,20 @@ RUN yum install -y epel-release \
     && chown eve:eve /home/eve/workspace \
     && pip install buildbot-worker==${BUILDBOT_VERSION}
 
+# Install Go tooling to build the Kubernetes Operators
+ENV GO_VERSION 1.12.7
+ENV OP_SDK_VERSION v0.9.0
+ENV GOROOT /usr/local/go
+ENV GOPATH /home/eve/.golang
+ENV GOCACHE /home/eve/workspace/.cache
+ENV PATH $GOPATH/bin:$GOROOT/bin:$PATH
+
+RUN curl -ORL https://dl.google.com/go/go${GO_VERSION}.linux-amd64.tar.gz      \
+    && tar xzvf go${GO_VERSION}.linux-amd64.tar.gz                             \
+    && rm go${GO_VERSION}.linux-amd64.tar.gz                                   \
+    && mv go /usr/local                                                        \
+    && curl -RLo /usr/bin/operator-sdk                                         \
+        https://github.com/operator-framework/operator-sdk/releases/download/${OP_SDK_VERSION}/operator-sdk-${OP_SDK_VERSION}-x86_64-linux-gnu \
+    && chmod +x /usr/bin/operator-sdk
```

Comment:
Consider checking the signature of the downloaded file?

Reply:
:+1: 
I'll add it (anyway I have to modify this Dockerfile, things aren't working yet)

Reply:
That may be harder than expected, because I don't have the maintainers public key, so I have to download it but to get the ID I need to parse the error message from `gpg`

> To download the key, use the following command, replacing $KEY_ID with the RSA key string provided in the output of the previous command:

:/

Reply:
Ugh, maybe file a TODO ticket but leave it for now.

Reply:
Tracked by #1444.